dev-util/codespell/codespell-2.1.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
I get
```
 * QA Notice: DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   have:     DISTUTILS_USE_SETUPTOOLS=rdepend
```

Reply:
Probably be good to mention both need `DISTUTILS_STRICT_ENTRY_POINTS=y` and "qa" in `PORTAGE_ELOG_CLASSES` to get this message.

Reply:
I was wondering, because I didn't get the warning. Didn't know about the `DISTUTILS_STRICT_ENTRY_POINTS=1` flag and only had qa in `PORTAGE_ELOG_CLASSES`.